deploy/src/main/resources/conf/app.conf

```diff
@@ -5,7 +5,7 @@
 #
 
 # If mode is DEVELOPMENT or STAGING, UI provides some extra tools
-environment = STAGING
+environment = DEVELOPMENT
```

Comment:
why this change?

Reply:
The development deployed instances were incorrectly showing STAGING.